File: JAVA/Fibbo.java
import java.util.*;
class Fibbo
{
    public static void main(String args[])
    {
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter number:");
        int n=sobj.nextInt();
int no1=0;
int no2=1;
int no3=0;
int sum=0;
for(int i=1;i<=n;i++)
{
    no3=no1+no2;
    System.out.println(no3);
    sum=sum+no3;
no1=no2;;
no2=no3;
}
System.out.println("Sum is"+sum);



    }
}

File: FEB/Fibbo.c
#include<stdio.h>
void Display(int No)
{
    int no1=0;
    int no2=1;
    int no3=0;
    printf("%d ",no1);
    printf("%d ",no2);
    for(int i=1;i<=No;i++)
    {
        no3=no1+no2;
        printf("%d ",no3);
        no1=no2;
        no2=no3;

    }
    
}

int main()
{
    int ino=0;
    printf("Enter the Number: ");
    scanf("%d",&ino);
    Display(ino);

    return 0;

}